packages/discovery-react-components/src/components/DiscoverySearch/DiscoverySearch.tsx

```diff
@@ -332,6 +336,12 @@ const DiscoverySearch: FC<DiscoverySearchProps> = ({
     overrideAutocompletionResults
   );
 
+  // Keep aggregationResults in sync with globalAggregationsResponseStore data
+  // to support it until deprecated in next major release
+  useDeepCompareEffect(() => {
+    setAggregationResults(globalAggregationsResponseStore.data);
```

Comment:
could you write a quick spec to validate this?

Reply:
Yes, updated with a test